src/OpenColorIO/Exception.cpp

```diff
@@ -31,24 +31,34 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 OCIO_NAMESPACE_ENTER
 {
   
-    Exception::Exception(const char * msg) throw()
-    : std::runtime_error(msg)
-    {}
+Exception::Exception(const char * msg)
+    :   std::runtime_error(msg)
```

Comment:
Just a style question. Further down in this PR, an initializer list has one space between ":" and first initializer, but here there are three. Should those be consistent?

Reply:
Yes, I will check and fix.

Reply:
A 'tab' but preserve the existing 'style' in the file you change. That's will be another 'rule' to define when we will put in place `clang-tidy`.

Reply:
Agreed